packets.js

```diff
@@ -341,7 +345,11 @@ answer.encode = function (a, buf, offset) {
   offset += name.encode.bytes
 
   buf.writeUInt16BE(types.toType(a.type), offset)
-  buf.writeUInt16BE(a.class === undefined ? 1 : a.class, offset + 2)
+
+  var klass = a.class === undefined ? 1 : a.class
+  if (a.flush) klass += FLUSH_MASK // the 1st bit of the class is the flush bit
```

Comment:
hmm.. yeah I guess you're right - it shows the intend more clearly, but the end result is the same since it's just an int. Should I change the `-=` above in a similar fashion?